src/slskd/Search/Types/Search.cs

```diff
@@ -34,7 +34,13 @@ public class Search
         public Guid Id { get; init; }
 
         [NotMapped]
-        public bool IsComplete => State.HasFlag(SearchStates.Completed);
+        public bool IsComplete =>
```

Comment:
same as below; searches (and transfers 🙃) are guaranteed to get the `Completed` flag regardless of how it ends, so checking for `Completed` is enough